OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -47,17 +48,18 @@ public class GravityBomb : IProjectile, ISync
 		readonly GravityBombInfo info;
 		readonly Animation anim;
 		readonly ProjectileArgs args;
+		readonly WVec acceleration;
 		[Sync] WVec velocity;
 		[Sync] WPos pos;
-		[Sync] WVec acceleration;
 
 		public GravityBomb(GravityBombInfo info, ProjectileArgs args)
 		{
 			this.info = info;
 			this.args = args;
 			pos = args.Source;
-			velocity = new WVec(WDist.Zero, WDist.Zero, -info.Speed);
-			acceleration = new WVec(WDist.Zero, WDist.Zero, info.Acceleration);
+			var facing = (args.PassiveTarget - pos).Yaw.Facing;
+			velocity = -info.Velocity.Rotate(WRot.FromFacing(facing));
```

Comment:
I guess you might be propagating the original `SourceActor` to the child projectiles (I don't see how else you could make that work), and really mean that the original actor's facing isn't relevant at the splitting time.  I guess it makes more sense to use the `args.Facing` instead and DIY the local -> world coordinate conversion.